sdl-core/src/main/scala/io/smartdatalake/workflow/action/DataFrameActionImpl.scala

```diff
@@ -352,7 +352,7 @@ abstract class DataFrameActionImpl extends ActionSubFeedsImpl[DataFrameSubFeed]
         subFeed.withMetrics(streamingMetrics.getMainInfos ++ sparkMetrics).asInstanceOf[DataFrameSubFeed]
       case _ =>
         // Auto persist if dataFrame is reused later
-        val preparedSubFeed = if (context.dataFrameReuseStatistics.contains((output.id, subFeed.partitionValues))) {
+        var preparedSubFeed = if (context.dataFrameReuseStatistics.contains((output.id, subFeed.partitionValues))) {
```

Comment:
why does this need to be a var now? I don t see any reassignement

Reply:
fixed